Don't call LifeCycleTracking for build < 14

matthiaswenz opened the pull request:
Fix for #46 
Shouldn't call any methods from LifeCycleTracking on build level < 14.

Commit:
Don't call LifeCycleTracking for build < 14

msftclas commented:
Hi __@ranterle__, I'm your friendly neighborhood Microsoft Pull Request Bot (You can call me MSBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no Contribution License Agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, MSBOT;

Commit:
Use Util method to determine feature availability

Commit:
Add more checks for feature availability

Another option would have been to make the LifeCycleTracking class public but I'd rather hide this in an implementation detail instead of exposing public API too early.
